SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -230,19 +256,16 @@ public class SwiftyTesseract {
   private func setEnvironmentVariable(_ variableName: TesseractVariableName, value: String) {
     setenv(variableName.rawValue, value, 1)
   }
-  
-  private func processPDF(images: [UIImage]) throws -> URL {
-    let filepath = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString)
-    
+
+  private func processPDF(images: [UIImage], at filepath: URL) throws {
+
     let renderer = try makeRenderer(at: filepath)
     
     defer {
       TessDeleteResultRenderer(renderer)
     }
     
     try render(images, with: renderer)
-    
-    return filepath.appendingPathExtension("pdf")
```

Comment:
The root cause of the failing tests is caused by this (which can be alleviated, but at the callsite). We _were_ returning a path with a `.pdf` file extension. When the renderer gets the path, it alters the file path to have the `.pdf` file extension.